pokedex_async_redux/lib/widgets/reusable_appbar.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_asyn_redux/utils/color_constants.dart';
+
+class CustomAppBar extends StatelessWidget implements PreferredSizeWidget {
+  final String title;
+
+  const CustomAppBar({required this.title, Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      backgroundColor: mainBgColor,
+      centerTitle: true,
+      title: Text(
+        title,
+        style: const TextStyle(
+          fontFamily: 'Staatliches',
+          fontSize: 35.0,
+          fontWeight: FontWeight.bold,
+        ),
+      ),
+    );
+  }
+
+  @override
+  Size get preferredSize => const Size.fromHeight(kToolbarHeight);
```

Comment:
do you still need this? if not kindly get rid as well on implementing or extending its class 🙂